### ISSUE TITLE
Fix sitemap format

### ISSUE THREAD

**Issue by MarcelCoding** [retypeapp/retype#125]
Add missing header:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
```
https://developers.google.com/search/docs/advanced/sitemaps/build-sitemap#xml

Google Search Console:
![grafik](https://user-images.githubusercontent.com/34819524/129410320-7f0f1f6b-a888-4f14-ad7b-d38d7a34b659.png)
The header is missing:
![grafik](https://user-images.githubusercontent.com/34819524/129410440-093da813-5b55-4ae7-b2e9-b679d2e344e0.png)

**Comment by MarcelCoding** [comment]
Somehow a fixed version is also not working

**Comment by geoffreymcgill** [comment]
Interesting.

When I view the [sitemap](https://retype.com/sitemap.xml.gz) for retype.com, I see the following:

```xml
<?xml version="1.0" encoding="utf-16"?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
  <url>
    <loc>https://retype.com/</loc>
    <lastmod>2021-08-09T19:11:33Z</lastmod>
    <priority>1</priority>
  </url>
```

Looks like we are generating as `encoding="utf-16"` but should be `encoding="UTF-8"`.

I see our original issue and sample did use `encoding="UTF-8"`, so I'm not sure how that `encoding="utf-16"` got in there. And it looks like it wasn't covered by our unit tests, so we'll have to fix that too.

**Comment by MarcelCoding** [comment]
i also found out that only the charset if from: UTF-16 -> UTF-8
my browser was hiding the header

**Comment by geoffreymcgill** [comment]
We found the issue and will fix right away. We will also validate the generated `sitemap.xml` format with Google Search Console and any other validation tools we can find, plus update our unit tests so they cover the sitemap content format. 

The fix will be included in the Retype `v1.9.0` release, which is scheduled for release on August 24 or 25.

Once your project is updated to `v1.9.0`, no other changes to your website will be required. It will just start working correctly.

**Comment by MarcelCoding** [comment]
I found a generatot witch is adding some more metadata:
```xml
<urlset
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xmlns:xhtml="http://www.w3.org/1999/xhtml"
  xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd"
  xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">	
```

https://www.check-domains.com/sitemap/

The sitemap generated by retype is working for bing.

**Comment by geoffreymcgill** [comment]
We will be adding the `xmlns:xhtml="http://www.w3.org/1999/xhtml"` attribute and the enhancement will also be included in the `v1.9.0` release. 

Thanks for the request!

**Comment by MarcelCoding** [comment]
Where is the source code? If you want I could open the pr.

**Comment by geoffreymcgill** [comment]
> Where is the source code? If you want I could open the pr.

Retype is a closed-source project.

**Comment by geoffreymcgill** [comment]
This sitemap issue has been fixed and will be included in the upcoming Retype `v1.9.0` release. Our goal is to have a `v1.9.0-preview` release available this week, with the final release early next week.
